js/src/components/contact-information/index.js

```diff
@@ -22,6 +29,50 @@ const description = __(
 const mcTitle = __( 'Enter contact information', 'google-listings-and-ads' );
 const settingsTitle = __( 'Contact information', 'google-listings-and-ads' );
 
+export function ContactInformationPreview() {
+	const phone = useGoogleMCPhoneNumber();
+
+	const handleEditClick = () => {
+		getHistory().push( getEditContactInformationUrl() );
+	};
+
+	let sectionContent;
+
+	if ( phone.loaded ) {
+		if ( phone.data.isValid ) {
+			sectionContent = (
+				<VerticalGapLayout size="overlap">
+					<PhoneNumberCard
+						isPreview
+						initEditing={ false }
+						onEditClick={ handleEditClick }
+					/>
```

Comment:
Dev note: follow the change of https://github.com/woocommerce/google-listings-and-ads/pull/917#discussion_r681154403 to pass `phone` as `phoneNumber` prop before merging this PR.

Reply:
Added by https://github.com/woocommerce/google-listings-and-ads/pull/921/commits/34e6f62afcd6e1554e3d6df84da5746ba11b0f49.